compair/models/file.py

```diff
@@ -1,7 +1,7 @@
 import mimetypes
 
 # sqlalchemy
-from sqlalchemy.orm import column_property
+from sqlalchemy.orm import column_property, joinedload
```

Comment:
added `joinedload` but not used?

Reply:
Ditto. Used in `get_active_or_404` and `get_active_by_uuid_or_404`.

But looking at it again, I wonder if these two functions are valid.  `File` is not `ActiveMixin` and has no `active` attribute.

Reply:
Yes, your right, we should just remove the functions (I might have been copied+pasted in when mass adding `get_active_or_404` and `get_active_by_uuid_or_404` since most models use active)